egs/swbd/s5c/local/reverberate/IR_stats.m

```diff
@@ -0,0 +1,348 @@
+function [rt,drr,cte,cfs,edt] = IR_stats(x, fs, varargin)
+%function [rt,drr,cte,cfs,edt] = IR_stats(filename, varargin)
```

Comment:
I see this file seems to be available online at
http://www.mathworks.com/matlabcentral/fileexchange/42566-impulse-response-acoustic-information-calculator/content/IR_stats.m
This could be a download, I guess- or a patched download if needed.  Anyway, at that location, it says the code is covered by the BSD license, which implies that we can redistribute it.